### DISCUSSION TITLE
Release 0.16 with fixed help text in multiprocess mode

### DISCUSSION THREAD

**Discussion by evgenymarkov** [prometheus/client_python#882] [General]
Hi! 👋 

When can we expect the release of version 0.16? I want to download the package with the fixed help text in multiprocess mode from PyPI instead of our private repository with patches.

Patch was shipped in https://github.com/prometheus/client_python/pull/804.

Thank you for your work! 🙏🏻

**Comment by evgenymarkov** [comment]
cc @csmarchbanks

**Comment by csmarchbanks** [comment]
Good reminder. I should be able to get a release out soon (this week at the latest, hopefully today or tomorrow). I didn't want to release over/just before the holidays and then forgot, sorry about that!

**Reply by csmarchbanks** [reply]
Released!

**Reply by evgenymarkov** [reply]
Thank you!
